certified-connectors/Haiilo/apiDefinition.swagger.json

```diff
@@ -4,8 +4,8 @@
   "definitions": {},
   "host": "smarp.smarpshare.com",
   "info": {
-    "description": "Smarp is a mobile-first employee communication platform to drive alignment, productivity and champion your brand. Connect to Smarp to manage your posts.",
-    "title": "Smarp",
+    "description": "Haiilo Advocacy is a mobile-first employee communication platform to drive alignment, productivity and champion your brand. Connect to Haiilo Advocacy to manage your posts.",
+    "title": "Haiilo Advocacy",
     "version": "1.0",
```

Comment:
@thanh-nguyen The "contact" property is required. Please see out template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json). 
```
    "contact": {
      "name": "Full name",
      "url": "Contact URL",
      "email": "Contact email address"
    }
```